aten/src/ATen/native/group_norm.cpp

```diff
@@ -181,7 +181,7 @@ Tensor group_norm(
 
   const Tensor kEmpty;
   auto memory_format = input.suggest_memory_format();
-  const auto& X = input.device().is_cpu() ?
+  const auto& X = input.device().is_cpu() || input.device().is_xpu() ?
```

Comment:
This is not following what cuda does here? That is expected?

Reply:
hi @albanD cuda doesn't support channels last here. Before only cpu is supported, and now we want to support it for xpu.